examples/limitrange/README.md

```diff
@@ -1,4 +1,147 @@
+more than that amount prior to admitting the pod in the cluster since that pod could never be scheduled by the
+system.  As a result, an operator has a desire to set the maximum amount of resource a single pod could consume
+on a system in order to increase the odds that a pod could fit on a node in the cluster.
+2. The cluster is shared by multiple communities in an organization to run both production and development
+workloads.  The cluster operator wants to restrict the amount of resource pods a development environment
+can consume in order to reserve capacity for the production workloads.  Development community workloads are
+separated from production community workloads by a Kubernetes namespace.  The development namespace accepts pods
+that consume no more than 512MB of memory, but the production namespace accepts pods that consume up to 8GB of
+memory.  To support each mode, the cluster operator applies separate limits to each namespace.
+3. Users may create a Pod which consume resources just below the capacity of a machine.  The left over space
+may be too small to be useful, but big enough for the waste to be costly over the entire cluster.  As a result,
+the cluster operator may want to set limits that a pod must consume at least 20% of the memory and cpu of their
+average node size in order to provide for more uniform scheduling and to limit waste.
+
+This example assumes you have a functional Kubernetes setup.
+
+This example demonstrates how limits can be applied to a Kubernetes namespace to control
+min/max resource limits per pod.  In addition, this example demonstrates how you can
+apply default resource limits to pods in the absence of an end-user specified value.
+
+Step 1: Create a namespace
+-----------------------------------------
+This example will work in a custom namespace to demonstrate the concepts involved.
+
+Let's create a new namespace called limit-example:
+
+```shell
+$ cluster/kubectl.sh create -f namespace.yaml
+$ cluster/kubectl.sh get namespaces
+NAME            LABELS             STATUS
+default         <none>             Active
+limit-example   <none>             Active
+```
+
+Step 2: Apply a limit to the namespace
+-----------------------------------------
+Let's create a simple limit in our namespace.
+
+```shell
+$ cluster/kubectl.sh create -f limits.yaml --namespace=limit-example
+```
+
+Let's describe the limits that we have imposed in our namespace.
+
+```shell
+Name:   limits
+Type    Resource  Min Max Default
+----    --------  --- --- ---
+Pod   memory    6Mi 1Gi -
+Pod   cpu   250m  2 -
+Container memory    6Mi 1Gi 100Mi
+Container cpu   250m  2 250m
+```
+
+In this scenario, we have said the following:
+
+1. A container in a pod may consume between 6Mi and 1Gi of memory.  If the container does not
+specify an explicit resource limit, each container in a pod will get 100Mi of memory.
+2. A container in a pod may consume between 250m and 2 cores of cpu.  If the container does
```

Comment:
nit: Might be worth clarifying the meaning of the 'm' suffix.  I assume that it's milli-cpu.